packages/app-desktop/gui/utils/NoteListUtils.ts

```diff
@@ -77,8 +77,8 @@ export default class NoteListUtils {
 						const note = await Note.load(noteIds[i]);
 						const newNote = Note.changeNoteType(note, type);
 						if (newNote === note) continue;
-						await Note.save(newNote, { userSideValidation: true });
-						eventManager.emit(EventName.NoteTypeToggle, { noteId: note.id });
+						const savedNote = await Note.save(newNote, { userSideValidation: true });
+						eventManager.emit(EventName.NoteTypeToggle, { noteId: note.id, note: savedNote });
```

Comment:
Based on a GitHub code search for `NoteTypeToggle` and `noteTypeToggle`, this event doesn't seem to be used (no listeners are created for it). It might make sense to remove it (though in a follow-up pull request).

---

Only certain properties are attached to the `NoteTypeToggle` event emitted by `toggleNoteType`:

https://github.com/laurent22/joplin/blob/50d08cd1787628c10b2d589097b23edf151a1038/packages/app-desktop/gui/MainScreen/commands/toggleNoteType.ts#L19-L25

As such, for consistency, it might make sense to only attach certain properties when emitting the event in `NoteListUtils.ts`.

Reply:
Yes looks like it could be removed then, I'm not sure what it was originally added. Maybe for a plugin? But indeed I can't find anything either on GitHub